control-myplaces/src/main/java/fi/nls/oskari/control/myplaces/handler/ExportMyPlacesFeaturesHandler.java

```diff
@@ -0,0 +1,64 @@
+package fi.nls.oskari.control.myplaces.handler;
+
+import fi.nls.oskari.annotation.OskariActionRoute;
+import fi.nls.oskari.control.ActionException;
+import fi.nls.oskari.control.ActionParameters;
+import fi.nls.oskari.domain.User;
+import fi.nls.oskari.log.LogFactory;
+import fi.nls.oskari.log.Logger;
+import fi.nls.oskari.service.ServiceException;
+import fi.nls.oskari.util.PropertyUtil;
+import org.json.JSONObject;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.*;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+@OskariActionRoute("ExportMyPlacesLayerFeatures")
+public class ExportMyPlacesFeaturesHandler extends MyPlacesFeaturesHandler {
+    private final static Logger LOG = LogFactory.getLogger(ExportMyPlacesFeaturesHandler.class);
+    private static final String PARAM_SRS = "srs";
+    private static final String PARAM_LAYER_ID = "categoryId";
+    private static final String PARAM_INDENT = "indent";
+    private static final String PARAM_PRETTIFY = "prettify";
+    private static final String FILE_EXT = "geojson";
+    private static final String FILE_TYPE = "application/json";
+    private static final int DEFAULT_INDENT = 2;
+
+    private static final DateTimeFormatter TIME_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+
+
+    @Override
+    public void handleGet(ActionParameters params) throws ActionException {
+        final User user = params.getUser();
+        final String srs = params.getHttpParam(PARAM_SRS, PropertyUtil.get("oskari.native.srs", "EPSG:4326"));
+        final String layerId = params.getRequiredParam(PARAM_LAYER_ID);
+        final int indent = params.getHttpParam(PARAM_INDENT, DEFAULT_INDENT);
+        final boolean prettify = params.getHttpParam(PARAM_PRETTIFY, true);
```

Comment:
I think we don't need a default indent value as if we remove it we don't need prettify. If the user gives an indent value between 1-8(?) we use that for indentation. I think we should make some sanity check for the value but if we do get one we can consider prettify == true.